desktop/src/main/java/bisq/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -221,6 +228,52 @@ public void initialize() {
         withdrawMemoTextField = addTopLabelInputTextField(gridPane, ++rowIndex,
                 Res.get("funds.withdrawal.memoLabel", Res.getBaseCurrencyCode())).second;
 
+        Tuple3<Label, InputTextField, ToggleButton> customFeeTuple = addTopLabelInputTextFieldSlideToggleButton(gridPane, ++rowIndex,
```

Comment:
You're reorganising 40 lines of code(more than 7lines i.e. readable). Where are the unit tests and/or end-to-end tests for this?

Reply:
That way we know it works all the time, and we also know when it gets accidentally broken by some other feature.

Reply:
I am just reusing most of the code that was already written for Settings. 
Unit tests were not available and I would prefer that someone more experienced than me could take that.